### PR TITLE
Move Product Reviews page classes to more specific namespace

### PR DESCRIPTION
**Pull request by dmagalhaes-godaddy**
## Summary

Moves the new classes to their own namespace.

### Story [MWC 5819](https://jira.godaddy.com/browse/MWC-5819)

Ref: https://github.com/woocommerce/woocommerce/pull/32763#pullrequestreview-968898718

## QA

- [ ] Code reviews
- ~[ ] Unit tests pass~ (will be addressed separately)

### CODE REVIEW

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php [hunk 1]
```diff
@@ -12,23 +12,23 @@
...
 class ReviewsListTableTest extends WC_Unit_Test_Case {
 
 	/**
-	 * Returns a new instance of the {@see ReviewsListTable} class.
+	 * Returns a new instance of the {@see \Automattic\WooCommerce\Internal\Admin\ProductReviews\ReviewsListTable} class.
 	 *
 	 * @return ReviewsListTable
 	 */
-	private function get_reviews_list_table() : ReviewsListTable {
+	private function get_reviews_list_table() : \Automattic\WooCommerce\Internal\Admin\ProductReviews\ReviewsListTable {
```
Comment (acastro1-godaddy) [inline plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]: I've seen a few class full names in this test class and in others too. Any specific reason for not importing that class and using its name only?

File: plugins/woocommerce/src/Internal/Admin/Loader.php [hunk 2]
```diff
@@ -9,6 +9,8 @@
 use Automattic\WooCommerce\Admin\Features\Features;
 use Automattic\WooCommerce\Admin\PageController;
 use Automattic\WooCommerce\Admin\PluginsHelper;
+use Automattic\WooCommerce\Internal\Admin\ProductReviews\Reviews;
```
Comment (acastro1-godaddy) [inline plugins/woocommerce/src/Internal/Admin/Loader.php:12]: This class has a few GH action failure messages, but apparently not related to this class' changes. Just noting, but I don't think we should fix them here unless directly related to the changes introduced.

**Review by acastro1-godaddy — COMMENTED** [review]
Code looks good, just left a couple of notes below.

File: plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php (hunk 1 above)
Comment (dmagalhaes-godaddy) [reply plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]: Not really, just how my IDE did it when I refactored the namespace. I'll tweak it.

Comment (dmagalhaes-godaddy) [reply plugins/woocommerce/tests/php/src/Internal/Admin/ProductReviews/ReviewsListTableTest.php]: Fixed in [4278670](https://github.com/godaddy-wordpress/woocommerce/pull/51/commits/42786707275a9736499c9d1a904ca65caeb9e98a).

File: plugins/woocommerce/src/Internal/Admin/Loader.php (hunk 2 above)
Comment (dmagalhaes-godaddy) [reply plugins/woocommerce/src/Internal/Admin/Loader.php:12]: I agree we shouldn't fix the code sniff issues in the Loader, as they are not introduced by this PR.

**Review by acastro1-godaddy — APPROVED** [review]
alright, considering that the code sniff issues are not introduced in this PR, I'm merging it.

**Pull request merged**